readme.md

```diff
@@ -93,6 +93,39 @@ prompt pure
 | **`PURE_GIT_DOWN_ARROW`**        | Defines the git down arrow symbol.                                                             | `⇣`            |
 | **`PURE_GIT_UP_ARROW`**          | Defines the git up arrow symbol.                                                               | `⇡`            |
 
+
+## Colors
+
+Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release/Zsh-Modules.html#The-zsh_002fzutil-Module) with a pattern of the form `:prompt:pure:$color_name` and style `color`. The color names, their default, and what part they affect are:
+- `exec_time` (yellow) - The execution time of the last command when exceeding `PURE_CMD_MAX_EXEC_TIME`
+- `git:arrow` (cyan) - For `PURE_GIT_UP_ARROW` and `PURE_GIT_DOWN_ARROW`
+- `git:branch` (242) - The name of the current branch when in a Git repository
+- `git:branch:cached` (red) - The name of the current branch when the data isn't fresh
+- `host` (242) - The hostname when on a remote machine
+- `path` (blue) - The current path, for example, `PWD`
+- `prompt:error` (red) - The `PURE_PROMPT_SYMBOL` when the previous command has *failed*
+- `prompt:success` (magenta) - The `PURE_PROMPT_SYMBOL` when the previous command has *succeded*
+- `user` (242) - The username when on remote machine
+- `user:root` (default) - The username when the user is root
```

Comment:
What does `default` mean? Should be clarified.

Reply:
`default` here means the terminal's default colour. Could also be called "uncolored"?

Reply:
I think `default` as a keyword is fine. We just need to explain in the docs what it means.

Reply:
Done.